### ISSUE TITLE
importMember with generic parameter fails

### ISSUE THREAD

**Issue by rkosafo** [fable-compiler/Fable#721]
### Description

When `importMember` is called with generic type, it fails.

### Repro code

    let omitBy: obj -> obj -> obj = importMember "lodash/fp" //works
    let omitBy<'t>: 't -> obj -> 't = importMember "lodash/fp" //fails

### Expected and actual results

Expected code to compile successful but it failed with `error FABLE: `importMember` must be assigned to a variable`

### Related information 

The following code works. It appears as long as it is obj, it is fine. Attempting to introduce generics fails the compilation

    module Fp =
        let isUndefined: obj -> bool = importMember fp
        let isNull: obj -> bool = importMember fp
        let omitBy: obj -> obj -> obj = importMember fp

    let inline omitBy<'t> (predicate:obj->bool) x : 't = (unbox Fp.omitBy) predicate x
    let _ = createObj ["a",box None; "b",box 5] |> omitBy Fp.isNull |> printfn "%A"

* fable-compiler version: 0.7.42
* fable-core version: 0.7.26
* Operating system: windows 10

**Comment by alfonsogarciacaro** [comment]
This is a bit tricky, maybe it can be fixed but the import expressions are designed to be used with values but a member like `omitBy<'t>` actually compiles as a method (because it has a type argument). You can actually use an explicit import expression here, but the generated code won't be probably as you expect:

```fsharp
let omitBy<'t> : 't -> obj -> 't = import "omitBy" "lodash/fp"
```

Probably the best option here would be to use an Import attribute:

```fsharp
open Fable.Core
open Fable.Core.JsInterop

[<Import("omitBy", from="lodash/fp")>]
let omitBy (x: 't) (o: obj): 't = jsNative
```

**Comment by rkosafo** [comment]
The explicit import expression did the trick.

    let omitBy<'t> : (obj->bool) -> 't -> 't = import "omitBy" fp 

translates to

      var omitBy = __exports.omitBy = function () {
        return omitBy_1;
      };

It is interesting that it is a function. When used, however, fable does the right thing.

    Fp.omitBy Fp.isUndefined x

becomes 

    Fp.omitBy()(Fp.isUndefined)(x);

It does solve it. Also, the `import` function feels more natural than the `Import` attribute :)

**Comment by alfonsogarciacaro** [comment]
Hi @rkosafo! I've made some corrections in the latest version (fable-compiler 0.7.43). Now you should be able to write the following:

```fsharp
let omitBy (x:'t) (o:obj): 't = importMember "lodash/fp"

// Please use the syntax above instead of
// let omitBy<'t>: 't -> obj -> 't = importMember "lodash/fp"
```

Can you update fable-compiler, give it a try and tell me how it goes?

**Comment by rkosafo** [comment]
let omitBy (_:obj) (_:'t): 't = importMember "lodash/fp"
    let omitBy (_:'t) (_:obj): 't = importMember "lodash/fp"

work but

    let omitBy (_:'t) (_:obj->bool): 't = importMember "lodash/fp"
    let omitBy (_:obj->bool) (_:'t): 't = importMember "lodash/fp"

fail with `error FABLE: `importMember` must be assigned to a variable`

I think the problem has to do with if any of the parameters is a function.


    let omitBy (_:obj->bool) (_:'t): 't = import "omitBy" "lodash/fp"

still works fine though.

**Comment by rkosafo** [comment]
Got some errors after some tests.

It looks like 

    let omitBy (_:obj->bool) (_:'t): 't = import "omitBy" "lodash/fp"

Gives the wrong translation. It is 

    var omitBy = __exports.omitBy = function (_arg2, _arg1) {
      return omitBy_1;
    };

Switched back to 

    let omitBy (_:obj) (_:'t): 't = import "omitBy" "lodash/fp"

which works

**Comment by alfonsogarciacaro** [comment]
Thanks for catching this @rkosafo! I've released fable-compiler 0.7.44 with a fix, can you please try?
